apps/site/lib/site_web/controllers/schedule/finder_api.ex

```diff
@@ -250,6 +257,10 @@ defmodule SiteWeb.ScheduleController.FinderApi do
   defp prepare_journeys_for_json(journeys) do
     journeys
     |> Enum.map(&destruct_journey/1)
+    |> Enum.filter(fn journey ->
+      get_in(journey, [:departure, :prediction, :time]) != nil ||
+        get_in(journey, [:departure, :schedule, :time]) != nil
```

Comment:
I think we're ok with journeys where the schedule is nil and the prediction is not (as seen in set_departure_time function)... but those would get filtered out here, right? I'm not so sure filtering here is the best idea.